CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Updated Developer guide to include docker build instructions ([#385](https://github.com/opensearch-project/opensearch-go/pull/385)
 - Test against version 2.9.0,2.10.0, run tests in all branches, change intergration tests to wait for OpenSearch to start ([#392](https://github.com/opensearch-project/opensearch-go/pull/392))
 - Makefile: use docker golangci-lint, run integration test on `.` folder, change coverage generation ([#392](https://github.com/opensearch-project/opensearch-go/pull/392)) 
+- test-unit.yml: added PR comment for unit test coverage report ([#407](https://github.com/opensearch-project/opensearch-go/pull/407))
```

Comment:
This change seems like uploading results to CodeCov, no?

Reply:
I am trying to understand the [output](https://app.codecov.io/gh/opensearch-project/opensearch-go/pull/407/indirect-changes#92a0d278e7f05c701b6cb6145a79652d-L432). Is it running it twice?